### PR TITLE
Add bottle example from IBM developerWorks wiki

### PR DESCRIPTION
**Pull request by kadler**
https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/IBM%20i%20Technology%20Updates/page/Sample%20web%20application%20with%20Python

### CODE REVIEW

File: non-wheel/bottle-example/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Active Jobs Dashboard
```
Comment (jordiwes) [inline non-wheel/bottle-example/README.md]: Correct title, pls :)

File: non-wheel/bottle-example/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Installing requisites
+ - Make sure you have installed 5733OPS Option 2, along with PTFs SI59051, SI60563, SI60564, and SI61963 (or subsequent PTFs)!
+   See [here](https://www.ibm.com/developerworks/community/wikis/home?lang=en#!/wiki/IBM%20i%20Technology%20Updates/page/Python%20PTFs) for the latest PTF numbers.
+ - ```pip3 install --no-index --find-links /QOpenSys/QIBM/ProdData/OPS/Python-pkgs/bottle --find-links /QOpenSys/QIBM/ProdData/OPS/Python-pkgs/ibm_db --find-links /QOpenSys/QIBM/ProdData/OPS/Python-pkgs/itoolkit -r requirements.txt```
+
+# Starting the server 
+python3 ./sample.py
+
+Point your web browser to http://&lt;systemname&gt;:9000
```
Comment (jordiwes) [inline non-wheel/bottle-example/README.md]: the route is /sample :)

File: non-wheel/bottle-example/README.md (hunk 1 above)
Comment (kadler) [reply non-wheel/bottle-example/README.md]: Done.

File: non-wheel/bottle-example/README.md (hunk 2 above)
Comment (kadler) [reply non-wheel/bottle-example/README.md]: and done.

**Pull request merged**